src/test/java/no/sikt/nva/scrapers/DublinCoreScraperTest.java

```diff
@@ -48,7 +48,7 @@ void shouldLogDcValuesThatAreNotUsedForScraping() {
                                                  + "gurba gurba gurba gurba gurba gurba gurba "
                                                  + "gurba gurba gurba gurba gurba (øæsdfadfåp)");
         var dublinCore = DublinCoreFactory.createDublinCoreWithDcValues(List.of(typeDcValue, expectedDcValuedLogged));
-        DublinCoreScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(null));
+        DublinCoreScraper.validateAndParseDublinCore(dublinCore, new BrageLocation(null), false);
```

Comment:
I det minste si hva den boolean'en betyr før den plasseres inn i testene, om du ikke går for å sette denne booleanen inn i konstruktøren.